### PR TITLE
Fix iteration 4 and create partial of iteration 5

### PR DESCRIPTION
**Pull request by yiranli624**
PR Template
## What is the change? 
- Viewing saved covers is now smaller in display. Also created "double click on picture, delete that chosen cover" function.
## What does it fix?
- It fixed the display size issue. 
-
 ## Is this a fix or a feature?
- Viewing saved cover is a fix, deleting function is a feature.
## Where should the reviewer start? 
- Line 8, 34, 117, 126-133.
## How should this be tested?
- Click "View Saved Covers" button, the display should be all much smaller compare with home cover view. 
- Double click on any saved cover picture, the selected cover will be removed from display, and can also be saved again.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ var newTaglineOne = document.querySelector('.tagline-1');
 var newTaglineTwo = document.querySelector('.tagline-2');
 var homePageLocation = document.querySelector('.home-view');
 var makeNewFormLocation = document.querySelector('.form-view');
-var viewSavedCoverLocation = document.querySelector('.saved-view');
+var viewSavedCoverLocation = document.querySelector('.saved-covers-section');
```
Comment (JSweet314) [inline src/main.js:8]: With this naming convention (`viewSaved...`), if I came across it anywhere other than here, I would expect this variable to be a function that I can call to view some thing, and not the html element itself. I would encourage calling it what it is - `savedCoversSectionElement` or something along those lines.

File: src/main.js [hunk 2]
```diff
@@ -30,19 +31,20 @@ makeNewFormButton.addEventListener('click', createNewCover);
...
-function randomHomePage(){
+function randomHomePage() {
   newCoverImage.src = covers[getRandomIndex(covers)];
   newTitle.innerText= titles[getRandomIndex(titles)];
   newTaglineOne.innerText = descriptors[getRandomIndex(descriptors)];
   newTaglineTwo.innerText = descriptors[getRandomIndex(descriptors)];
 }
 function showNewRandomCover() {
-  currentCover = new Cover(covers[getRandomIndex(covers)], titles[getRandomIndex(titles)], descriptors[getRandomIndex(descriptors)], descriptors[getRandomIndex(descriptors)] );
+  currentCover = new Cover(covers[getRandomIndex(covers)], titles[getRandomIndex(titles)], descriptors[getRandomIndex(descriptors)], descriptors[getRandomIndex(descriptors)]);
```
Comment (JSweet314) [inline src/main.js:47]: This line is difficult to read, consider putting the arguments passed to the Cover class on new lines. In general, most codebases I see enforce a max line length of 80-100 characters.

File: src/main.js [hunk 3]
```diff
@@ -51,14 +53,13 @@ function showNewRandomCover() {
...
-  viewSavedCoverLocation.classList.add('hidden');
+  viewSavedViewLocation.classList.add('hidden');
   randomCoverButton.classList.add('hidden');
   saveCoverButton.classList.add('hidden');
   homeButton.classList.remove('hidden');
 }
 function viewSavedCovers() {
-  viewSavedCoverLocation.classList.remove('hidden');
-  console.log(savedCovers);
+  viewSavedViewLocation.classList.remove('hidden');
```
Comment (JSweet314) [inline src/main.js:62]: Appears there is a lot of repetition around hiding and displaying elements - consider whether it makes sense to refactor and dry this up.

File: src/main.js [hunk 4]
```diff
@@ -122,4 +123,12 @@ function displaySavedCovers() {
     </section>`;
   }
 }
+function removeCover() {
+  for (var i = 0; i < savedCovers.length; i++) {
```
Comment (JSweet314) [inline src/main.js:127]: This function iterates through the entire array, regardless of whether the saved cover in question is found before reaching the last element in the array. How might you prevent that?

File: src/main.js [hunk 5]
```diff
@@ -109,11 +110,11 @@ function saveCover() {
...
 }
 function displaySavedCovers() {
-  viewSavedCovers() 
+  viewSavedCovers();
   viewSavedCoverLocation.innerHTML = '';
   for (var i = 0; i < savedCovers.length; i++) {
-    viewSavedCoverLocation.innerHTML +=
-    `<section class = "main-cover">
+    viewSavedCoverLocation.innerHTML += 
+    `<section class = "mini-cover">
```
Comment (JSweet314) [inline src/main.js:117]: html convention is to run attributes and values together. I.e. `class="something"`, as opposed to `class = "something"`

File: src/main.js [hunk 6]
```diff
@@ -67,13 +68,13 @@ function viewSavedCovers() {
...
+  viewSavedViewLocation.classList.add('hidden');
   randomCoverButton.classList.remove('hidden');
   saveCoverButton.classList.remove('hidden');
   homeButton.classList.add('hidden');
   homePageLocation.classList.remove('hidden');
 }
-function storeUserData(event) {
+function storeUserData() {
   currentCover = new Cover (userCover.value, userTitle.value, userDesc1.value, userDesc2.value);
   event.preventDefault();
```
Comment (JSweet314) [inline src/main.js:79]: If you are preventing default behavior, best to move this to the top line of the function for clarity and to ensure the default behavior is actually prevented

**Review by JSweet314 — COMMENTED** [review]
Overall, looks good.

I found the naming conventions to be a bit odd. In general, verbs should be associated with functions, but in some cases you have verbs referring to HTML elements or other things that don't "do" anything. My advice here - call a spade a spade. For example, if the variable represents an HTML element, it is useful to have `element` or `htmlElement` in the variable name. 

Double check your html syntax.

Consider some of the comments below re: refactoring/DRYing up the code where it makes sense. 

Keep up the good work!

File: src/main.js (hunk 4 above)
Comment (yiranli624) [reply src/main.js:127]: by adding an if condition. So it wouldn't start iterating unless the condition is satisfied.

Comment (JSweet314) [reply src/main.js:127]: I don’t think you need another if statement. Look into how you might be able to _break_ out of a loop

**Pull request closed without merging**